sdk/oracledatabase/azure-resourcemanager-oracledatabase/CHANGELOG.md

```diff
@@ -1,14 +1,21 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0 (2024-07-12)
 
-### Features Added
+- Azure Resource Manager OracleDatabase client library for Java. This package contains Microsoft Azure SDK for OracleDatabase Management SDK.  Package tag package-2023-09-01. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
 ### Breaking Changes
 
-### Bugs Fixed
+#### `models.DbSystemShapeProperties` was modified
+
+* `int availableCoreCount()` -> `java.lang.Integer availableCoreCount()`
 
-### Other Changes
+#### `models.DnsPrivateZoneProperties` was modified
+
+* `int serial()` -> `java.lang.Integer serial()`
+* `boolean isProtected()` -> `java.lang.Boolean isProtected()`
+
+#### `models.DnsPrivateViewProperties` was modified
```

Comment:
This one seems not complete?

Reply:
Thanks, missed that one... https://github.com/Azure/azure-sdk-for-java/pull/40915/files

Seems only the property required -> non-required change.